src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://airbnb.com.us"
+      method="post"
+      class="form"
+      data-qa="big"
+    >
```

Comment:
Sorry, one more line as "action" has to be from a new line. 
Check other tags as well 
```suggestion
    <form 
      action="https://airbnb.com.us"
      method="post"
      class="form"
      data-qa="big"
    >
```